pom.xml

```diff
@@ -17,6 +17,16 @@
               <mainClass>com.github.cypher.Main</mainClass>
           </configuration>
       </plugin>
+
```

Comment:
As specified in the commit message those changes are aimed at compiling the project with JDK 1.8 instead of 1.5 as predefined. The plugin configuration added to this file helps us do that without manually specifying the JDK flags at compile time.